manager/config/integration-tests/manager_patch.yaml

```diff
@@ -12,26 +12,22 @@ spec:
   template:
     spec:
       containers:
+      - name: integration-tests
+        image: alpine/socat:1.0.5
+        command:
+        - socat
+        - TCP4-LISTEN:5000,fork
+        - TCP4:kind-registry:5000
```

Comment:
What is the purpose of an additional container?

Reply:
As described in the issue and the PR head, it makes the registry reachable via localhost.

Reply:
ok, got it